docs/docsite/rst/dev_guide/developing_modules_best_practices.rst

```diff
@@ -34,7 +34,7 @@ General guidelines & tips
 * Each module should be self-contained in one file, so it can be auto-transferred by ``ansible-base``.
 * Module name MUST use underscores instead of hyphens or spaces as a word separator. Using hyphens and spaces will prevent ``ansible-base`` from importing your module.
 * Always use the ``hacking/test-module.py`` script when developing modules - it will warn you about common pitfalls.
-* If you have a local module that returns facts specific to your installations, a good name for this module is ``site_facts``.
+* If you have a local module that returns facts specific to your installations, a good name for this module is ``site_info``.
```

Comment:
We should use "info" or "facts" consistently:
```suggestion
* If you have a local module that returns information specific to your installations, a good name for this module is ``site_info``.
```

Reply:
good spot! thanks